systems/frontier_user_guide.rst

```diff
@@ -233,54 +233,48 @@ And here is the output from the script:
 
 AMD GPUs
 ========
-.. note::
-
-   This section on AMD GPUs is still a work in progress.
-
 
-Each Frontier node contains 4x AMD Instinct MI250X accelerators. Each MI250X has 2
-Graphical Compute Dies (GCD) for a total of 8 GCDs per node. Each GCD can be treated as
-its own independent GPU.
+The AMD Instinct MI200 is built on advanced packaging technologies
+enabling two Graphic Compute Dies (GCDs) to be integrated
+into a single package in the Open Compute Project (OCP) Accelerator Module (OAM)
+in the MI250 and MI250X products.
+Each GCD is build on the AMD CDNA 2 architecture.
+A single Frontier node contains 4 MI250X OAMs for the total of 8 GCDs.
 
 .. note::
 
-    **TERMINOLOGY:**
-
-    The 8 GCDs contained in the 4 MI250X will show as 8 separate GPUs according to Slurm,
-    ``ROCR_VISIBLE_DEVICES``, and the ROCr runtime, so from this point forward in the
-    Frontier guide, we will simply refer to the GCDs as GPUs.
+    The Slurm workload manager and the ROCr runtime treat each GCD as a separate GPU
+    and visibility can be controlled using the ``ROCR_VISIBLE_DEVICES`` environment variable.
+    Therefore, from this point on, the Frontier guide simply refers to a GCD as a GPU.
 
-Each GPU has a peak performance of 26.5 TFLOPS (double-precision), 110 compute units, and
-64 GB of high-bandwidth memory (HBM2) which can be accessed at a peak of 1.6 TB/s. The 2
-GPUs on an MI250X are connected with [4x] GPU-to-GPU Infinity Fabric links for a total
-bandwidth of 200+200 GB/s . Consult the diagram in the  section :ref:`frontier-nodes` for
-information on how the accelerators connect to each other, to the CPU, and to the network.
+Each GPU contains 110 Compute Units (CUs) grouped in 4 Compute Engines (CEs).
```

Comment:
HI @jakurzak , one thing I wanted to clarify is the fact that each GCD has 110 CUs but the diagram you added shows 112 CUs in each GCD. Can you clarify if there are two that are inactive and which ones they are?

Reply:
I clarified that there are 112 physical CUs and two are disabled.